opt/TEMPLATE

```diff
@@ -0,0 +1,43 @@
+# Example experiment configuration template
```

Comment:
This file doesn't feel necessary anymore since user's won't be creating their own rose-suite config